### PR TITLE
fix: make predicate value support experimental signals

### PR DESCRIPTION
**Pull request by kanitw**
Merge https://github.com/vega/vega-lite/pull/6083 first

### CODE REVIEW

File: src/predicate.ts [hunk 1]
```diff
@@ -117,7 +118,7 @@ export interface FieldRangePredicate extends FieldPredicateBase {
    * @maxItems 2
    * @minItems 2
    */
-  range: (number | DateTime | null)[];
+  range: (number | DateTime | null | SignalRef)[];
```
Comment (domoritz) [inline src/predicate.ts]: Should this be `(number | DateTime | null | SignalRef)[]` or `(number | DateTime | null)[] | SignalRef`?

Comment (kanitw) [reply src/predicate.ts]: Ah yes, technically both.

**Comment by kanitw** [comment]
I added signal support for range, take a look again?

**Comment by kanitw** [comment]
I guess this should be fine.

**Pull request merged**